app/controllers/refinery/blog/admin/categories_controller.rb

```diff
@@ -4,8 +4,7 @@ module Admin
       class CategoriesController < ::Refinery::AdminController
 
         crudify :'refinery/blog/category',
-                :include => [:translations],
-                :order => 'title ASC'
```

Comment:
Why do we remove the order? can't we use it?

Reply:
You're right @bricesanchez. As the `title` column was not available anymore and the order was not absolutely necessary to me, I removed it.
But I agree with you, I included it again in a new commit